sdk/python/sawtooth_sdk/processor/core.py

```diff
@@ -239,6 +248,10 @@ def _unregister(self):
                         vce)
 
     def start(self):
+        """Connects the TransactionProcessor to a Validator and starts
```

Comment:
Same question as above ("transaction processor" vs "TransactionProcessor).  Also, should it be "validator" instead of "Validator"?

Reply:
I was hoping the doc generator would convert those to links, but it didn't :(

Reply:
In a perfect world, this ought to be possible.  I wonder if Shawn knows how to make it work?